_includes/hero.html

```diff
@@ -1,6 +1,6 @@
 <div class="hero">
   <div class="container">
     <img class="logo" src="{{ site.baseurl }}/img/logo.png" alt="Istio">
-    <h1 class="hero-lead">A service mesh for polyglot microservices</h1>
+    <h1 class="hero-lead">An open platform to connect, manage, and secure microservices</h1>
```

Comment:
Can we just say "services"? What if I have a big monolithic service in my cluster, can I not use Istio?